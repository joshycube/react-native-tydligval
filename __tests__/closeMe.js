import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import CloseMe from "../svg/closeMe";

describe("closeMe", () => {
  test("renders the SVG correctly", () => {
    const shallow = new ShallowRenderer();
    shallow.render(<CloseMe />);
    const result = shallow.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
