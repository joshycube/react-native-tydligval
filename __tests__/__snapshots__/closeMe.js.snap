// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`closeMe renders the SVG correctly 1`] = `
<Svg
  height="25"
  preserveAspectRatio="xMidYMid meet"
  width="25"
>
  <G>
    <G
      fill="#ffffff"
      transform="translate(-454.000000, -1070.000000)"
    >
      <G
        transform="translate(462.000000, 1078.000000) scale(-1, 1) translate(-462.000000, -1078.000000) translate(454.000000, 1070.000000)"
      >
        <Path
          d="M15.7051985,2.98855196 C16.0984858,2.59526482 16.0984858,1.95603851 15.7038853,1.56143816 L14.4385645,0.296117566 C14.0439641,-0.0984827851 13.4047377,-0.0984827851 13.0114504,0.294800997 L8.71290191,4.59334873 C8.31961469,4.98663588 7.68038826,4.98663588 7.28710104,4.59334873 L2.98855252,0.294800997 C2.59526529,-0.0984861523 1.95603887,-0.0984861523 1.56143845,0.296117566 L0.296117621,1.56143816 C-0.0984828033,1.95603851 -0.0984828033,2.59526482 0.294801052,2.98855196 L4.59334958,7.2870997 C4.9866368,7.68038685 4.9866368,8.31961315 4.59334958,8.7129003 L0.294801052,13.011448 C-0.0984861704,13.4047352 -0.0984861704,14.0439615 0.296117621,14.4385618 L1.56143845,15.7038824 C1.95603887,16.0984828 2.59526529,16.0984828 2.98855252,15.705199 L7.28710104,11.4066513 C7.68038826,11.0133641 8.31961469,11.0133641 8.71290191,11.4066513 L13.0114504,15.705199 C13.4047377,16.0984862 14.0439641,16.0984862 14.4385645,15.7038824 L15.7038853,14.4385618 C16.0984858,14.0439615 16.0984858,13.4047352 15.7051985,13.011448 L11.40665,8.7129003 C11.0133628,8.31961315 11.0133628,7.68038685 11.40665,7.2870997 L15.7051985,2.98855196 Z"
        />
      </G>
    </G>
  </G>
</Svg>
`;
